com.ibm.wala.cast.java.ecj/src/com/ibm/wala/cast/java/translator/jdt/JDTJava2CAstTranslator.java

```diff
@@ -287,16 +287,13 @@ public CAstEntity translateToCAst() {
 
     private final T fSourcePosition;
 
-    private final Set<CAstAnnotation> annotations;
-    
     public ClassEntity(ITypeBinding jdtType, String name, Collection<CAstQualifier> quals, Collection<CAstEntity> entities,
```

Comment:
We should also delete the unused constructor parameter and fix up callers.  Same for all changes in commit 934f8f524f228ecd5c8de7c6d9dac6a9efca4b7c where relevant